.github/workflows/verify-binary-installation.yml

```diff
@@ -0,0 +1,54 @@
+name: 'Install Dashboards with Plugin via Binary'
+
+on: [push, pull_request]
+env:
+  OPENSEARCH_VERSION: '3.0.0'
```

Comment:
i have backported this [commit](https://github.com/opensearch-project/dashboards-notifications/commit/d1a3c8d59f989fb7326d31cba3485162846872db ) from the main to 2.x branch, I see OPENSEARCH_VERSION is 3.0.0' is it supposed to be 3.0.0 fro 2.x, because 2.x should be tested again 2.14/2.x osd version

Reply:
`OPENSEARCH_VERSION` will overwrite by line 27 `echo "OPENSEARCH_VERSION=$opensearch_version" >> $GITHUB_ENV`, that's should not be a issue.